src/ActionBar/ActionBar.js

```diff
@@ -55,7 +57,7 @@ const ActionBar = React.forwardRef(({
         actionClassName
     );
 
-    const HeadingTag = `h${headingLevel}`;
+    const titleStyle = headingStyle || headingLevel;
```

Comment:
Kind of minor, but we could simplify this a bit if we just passed `levelStyle={headingStyle}>` into `<Title />` since the same logic happens there if it's not defined
```js
const levelClass = levelStyle ? `h${levelStyle}` : `h${level}`;
```